lib/services/local/resource.go

```diff
@@ -446,6 +446,14 @@ func itemToLocalAuthSecrets(items userItems) (*types.LocalAuthSecrets, error) {
 		}
 		auth.MFA = append(auth.MFA, &d)
 	}
+	if items.webauthnLocalAuth != nil {
+		wal := &types.WebauthnLocalAuth{}
+		err := json.Unmarshal(items.webauthnLocalAuth.Value, wal)
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
+		auth.Webauthn = wal
+	}
 
 	// DELETE IN 7.0: these items are migrated to items.mfa on 6.0 first
```

Comment:
Should we remove this now while you're familiar with this code path?

Reply:
Yes, a cleanup is on my radar, but I think I'll do it in a separate PR.